client/src/js/references/components/Detail/Targets/Item.js

```diff
@@ -1,27 +1,37 @@
 import React from "react";
 import styled from "styled-components";
-import { BoxGroupSection, Icon, Flex, FlexItem } from "../../../../base";
+import { BoxGroupSection, Icon } from "../../../../base";
 
-const NameIcon = styled.div`
+const TargetItemHeader = styled.h3`
+    align-items: center;
     display: flex;
-    justify-content: space-between;
+    font-size: 14px;
+    font-weight: normal;
+    margin: 3px 0 6px;
+
+    span {
+        margin-left: auto;
+
+        i:not(first-child) {
+            margin-left: 3px;
+        }
+    }
+`;
+
+const TargetItemDescription = styled.p`
+    font-style: ${props => (props.description ? "normal" : "italic")};
+    margin: 0;
 `;
 
-export const TargetItem = ({ name, description, onEdit, onRemove }) => {
-    return (
-        <BoxGroupSection>
-            <NameIcon>
-                {name}
-                <FlexItem grow={1} shrink={1}>
-                    <Flex alignItems="center" className="pull-right">
-                        <Icon name="edit" bsStyle="warning" tip="Modify" onClick={onEdit} />
-                        <FlexItem pad>
-                            <Icon name="trash" bsStyle="danger" tip="Remove" onClick={onRemove} />
-                        </FlexItem>
-                    </Flex>
-                </FlexItem>
-            </NameIcon>
-            {description}
-        </BoxGroupSection>
-    );
-};
+export const TargetItem = ({ canModify, description, name, onEdit, onRemove }) => (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['canModify' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4993528)